### DISCUSSION TITLE
" Support for the experimental syntax 'jsx' isn't currently enabled" while trying to move from babel to swc for my NextJS project

### DISCUSSION THREAD

**Discussion by sahanaravi** [swc-project/swc#7947] [Q&A]
I am trying to move from babel to swc for my NextJS (13) project and keep getting - _Support for the experimental syntax 'jsx' isn't currently enabled .Add @babel/preset-react (https://github.com/babel/babel/tree/main/packages/babel-preset-react) to the 'presets' section of your Babel config to enable transformation.
If you want to leave it as-is, add @babel/plugin-syntax-jsx (https://github.com/babel/babel/tree/main/packages/babel-plugin-syntax-jsx) to the 'plugins' section to enable parsing._ How can I achieve this using swc?

**Comment by kwonoj** [comment]
This should go next.js question with your next.config.js.
